azure-pipelines-release.yml

```diff
@@ -0,0 +1,133 @@
+trigger: none
+pr: none
+
+# Variables are prefixed with a _, as they should be defined by the job on
+# Azure Pipelines (without the _). Sadly, if they are defined here without
+# _, they overwrite anything you set during job queue.
+variables:
+  # If this is false, not all targets are triggered. For example:
+  # both the NSIS installer for Windows as the creation of debs only work
+  # for releases. Not for any other type of binary.
+  _IsStableRelease: false
+
+jobs:
+- job: windows
+  displayName: 'Windows'
+  pool:
+    vmImage: 'VS2017-Win2016'
+
+  strategy:
+    matrix:
+      Win32:
+        BuildPlatform: 'Win32'
+        BundlePlatform: 'win32'
+      Win64:
+        BuildPlatform: 'x64'
+        BundlePlatform: 'win64'
+
+  steps:
+  - template: azure-pipelines/release-detect-version.yml
+  - template: azure-pipelines/windows-dependencies.yml
+  - template: azure-pipelines/windows-dependency-zip.yml
+  - template: azure-pipelines/windows-dependency-nsis.yml
+    parameters:
+      condition: eq(variables['IsStableRelease'], true)
+  - template: azure-pipelines/windows-build.yml
+    parameters:
+      BuildPlatform: $(BuildPlatform)
+  - bash: |
+      set -ex
+      make -f Makefile.msvc bundle_pdb bundle_zip PLATFORM=$(BundlePlatform) BUNDLE_NAME=openttd-$(Build.BuildNumber)-windows-$(BundlePlatform)
+    displayName: 'Create bundles'
+  - bash: |
+      set -ex
+      # NSIS will be part of the Hosted image in the next update. Till then, we set the PATH ourself
+      export PATH="${PATH}:/c/Program Files (x86)/NSIS"
+      make -f Makefile.msvc bundle_exe PLATFORM=$(BundlePlatform) BUNDLE_NAME=openttd-$(Build.BuildNumber)-windows-$(BundlePlatform)
+    displayName: 'Create installer bundle'
+    condition: and(succeeded(), eq(variables['IsStableRelease'], true))
+  - template: azure-pipelines/release-bundles.yml
+
+- job: source
+  displayName: 'Source'
+  pool:
+    vmImage: 'ubuntu-16.04'
+
+  steps:
+  - template: azure-pipelines/release-detect-version.yml
+  - script: |
+      set -ex
+      ./findversion.sh | cut -f 1 -d$'\t' > .version
```

Comment:
maybe instead of running `findversion.sh` twice, `.version` could be generated from `cat`ting `.ottdrev` ?